go.mod

```diff
@@ -155,3 +155,5 @@ require (
 	k8s.io/klog v1.0.0 // indirect
 	tags.cncf.io/container-device-interface v0.6.2 // indirect
 )
+
+replace github.com/opencontainers/runtime-spec => github.com/opencontainers/runtime-spec v1.1.0
```

Comment:
crun on debian is not updated to a version that will work, so in order to move forward we are not advancing the runtime-spec.  As soon as debian moves forward this can be removed.

This will not effect the effect the eventual podman build PR